package/koreader/koreader

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
A bit nitpicky, but using bash for running another command seems overkill to me.

Reply:
Just looked up the links and it wouldn't make sense to change this to `/bin/sh` since they default to bash anyway.